src/Tribe/Validate.php

```diff
@@ -227,6 +227,20 @@ public function positive_int() {
 		}
 
 		/**
+		 * validates a field as being an integer
+		 *
+		 * @return stdClass validation result object
+		 */
+		public function int() {
```

Comment:
I was trying to adhere to the naming convention of the other validation method: `positive_int()`. I just assume keep it as `int()`